recipes/pyamg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+    name: pyamg
+    version: "3.0.2"
+
+source:
+    git_url: https://github.com/pyamg/pyamg.git
+    git_tag: v3.0.2
```

Comment:
Sure. Take a look at [this]( https://github.com/conda-forge/webcolors-feedstock/blob/master/recipe/meta.yaml#L1-L10 ) as an example.